roles/user-liaison.md

```diff
@@ -0,0 +1,10 @@
+# User Liaison
+The user liaison is a role within the Saturn team that addresses issues that block only a few users on prod (whereas on-call addresses issues that block many users on prod) The only requirements to participate in the user liaison process are having a development setup. The user liaison is responsible for watching #saturn and #workbench-userliaison (the userliaison channel is private. If you want access for user liaison duties, message a member of the comms team)
```

Comment:
This looks good but just maybe a slight re-wording about requirements being having a development setup and access to the user liaison channel:

The only requirements to participate in the user liaison process are having a development setup "and having access to the user liaison channel on slack. This channel is private so to take on user liaison duties you will need to message a member of the comms team for access. The user liaison is responsible for watching #saturn and #workbench-userliaison for any reports on bugs on our services."